subscriptions.go

```diff
@@ -55,6 +55,7 @@ type Subscription struct {
 	PONumber               string              `xml:"po_number,omitempty"`
 	NetTerms               NullInt             `xml:"net_terms,omitempty"`
 	SubscriptionAddOns     []SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	PendingSubscription    PendingSubscription `xml:"pending_subscription,omitempty"`
```

Comment:
Updated in [1124e2](https://github.com/blacklightcms/recurly/pull/17/commits/2412af5d93f1b0b21c9af39ede36b72af31124e2).